javascript/javascript1/week4/classExercises/creditCardInfo.js

```diff
@@ -0,0 +1,48 @@
+function isValid(arr) {
```

Comment:
The task didn't require more validation than the basic type through the first card number, but you've went the extra mile. Well done!